### DISCUSSION TITLE
Patch release containing fix for multiple type attributes?

### DISCUSSION THREAD

**Discussion by mrp67** [dynamoose/dynamoose#1103] [Q&A] [answered]
Would it be possible to have a patch release containing the fix for multiple type attributes (https://github.com/dynamoose/dynamoose/commit/e8edc73da0714090b84facf3d38e868295f4d551)?

Since everything is bundled into a dist folder, installing the package by referencing a commit doesn't work. Instead, I have a bash script copying the fixed files into my node_modules. This approach is very error-prone. A patch release containing the fix would be a huge help!

**Comment by fishcharlie** [comment, marked as answer]
@mrp67 I'm planning to release v2.6.0 by end of this weekend at the latest.

However, running `npm i git://github.com/dynamoose/dynamoose.git#e8edc73` works perfectly. I'm unsure why this option doesn't work for you. There is no reason why you can't reference a commit or install the `master` branch.

**Reply by mrp67** [reply]
@fishcharlie when doing that, there was no dist folder, and so there were no source files. Just a README and a few other files from the root of the repo.

**Reply by fishcharlie** [reply]
@mrp67 Is it throwing some type of error?

Here is what I got when I tried to run that command:

```
charliefish ~ % mkdir tmp10
charliefish ~ % cd tmp10
charliefish tmp10 % npm i git://github.com/dynamoose/dynamoose.git#e8edc73
npm WARN saveError ENOENT: no such file or directory, open '/Users/charliefish/tmp10/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/Users/charliefish/tmp10/package.json'
npm WARN tmp10 No description
npm WARN tmp10 No repository field.
npm WARN tmp10 No README data
npm WARN tmp10 No license field.

+ dynamoose@2.5.0
added 19 packages from 104 contributors and audited 19 packages in 41.139s

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

charliefish tmp10 % ls
node_modules		package-lock.json
charliefish tmp10 % cd node_modules 
charliefish node_modules % ls
aws-sdk			ieee754			source-map
base64-js		isarray			source-map-support
buffer			jmespath		url
buffer-from		punycode		uuid
dynamoose		querystring		xml2js
events			sax			xmlbuilder
charliefish node_modules % cd dynamoose 
charliefish dynamoose % ls
CHANGELOG.md	LICENSE		README.md	dist		package.json
charliefish dynamoose % cd dist
charliefish dist % ls
Condition.d.ts			ModelStore.js
Condition.js			ModelStore.js.map
Condition.js.map		Populate.d.ts
Document.d.ts			Populate.js
Document.js			Populate.js.map
Document.js.map			Schema.d.ts
DocumentRetriever.d.ts		Schema.js
DocumentRetriever.js		Schema.js.map
DocumentRetriever.js.map	Serializer.d.ts
Error.d.ts			Serializer.js
Error.js			Serializer.js.map
Error.js.map			Transaction.d.ts
General.d.ts			Transaction.js
General.js			Transaction.js.map
General.js.map			aws
Internal.d.ts			index.d.ts
Internal.js			index.js
Internal.js.map			index.js.map
Model				logger
ModelStore.d.ts			utils
```

**Reply by mrp67** [reply]
@fishcharlie it's all there now. Not sure what was going on when I tried it yesterday. Thanks for your help and looking forward to the 2.6.0 release!
